arch/risc-v/src/mpfs/Kconfig

```diff
@@ -700,6 +700,14 @@ config MPFS_PHYINIT
 	---help---
 		call mpfs_phy_boardinitialize() on init
 
+config MPFS_PHY_RX_TIMEOUT_WA
+	int "RX restart timeout workaround"
+	default 0
```

Comment:
@jlaitine what about adding a range field to avoid user putting a very long time value?